gensrc/thrift/AgentService.thrift

```diff
@@ -361,6 +361,9 @@ struct TTabletMetaInfo {
 
 struct TUpdateTabletMetaInfoReq {
     1: optional list<TTabletMetaInfo> tabletMetaInfos
+    // for update lake tablet meta
+    2: optional TTabletType tablet_type
+    3: optional i64 txn_id
```

Comment:
Most RPCs for shared-storage tables have migrated to brpc/Protobuf, except for a few like create tablet and schema change. Do you think the workload is acceptable to also migrate update metadata to Protobuf?

Reply:
I mean, instead of reusing the existing Thrift structure here, how about adding a new Protobuf structure and RPC service in the lake_service.proto?

Reply:
I'm uncertain how much workload is involved. If it's significant, then continuing to use the Thrift protocol here is acceptable.

Reply:
Ok, I will move the thrfit protocol to lake_service in another pr